tests/file-size-lint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+FILE='build/css/build.min.css'
+if [ ! -f $FILE ]; then
+  echo 'ERROR: Build file could not be found or is not readable. Please build first with `./run build`'
+  exit 1
+fi
+
+THRESHOLD=400000
+CURRENTSIZE=$(du -b $FILE | cut -f 1)
+MESSAGE="Your generated CSS is ${CURRENTSIZE} with a threshold of ${THRESHOLD}"
+if [ $CURRENTSIZE -lt $THRESHOLD ]; then
+  STATUS='SUCCESS'
```

Comment:
Why not simply `echo` and exit directly in each of these cases? I know you'll be duplicating the message, but arguably those messages should be slightly different, given the radically different results:

``` bash
if [ ${CURRENTSIZE} -gte ${THRESHOLD} ]; then
  echo "ERROR: Your CSS is over ${THRESHOLD} (it's ${CURRENTSIZE})"
  exit 1
fi

echo "Success: your generated CSS is ${CURRENTSIZE} (under ${THRESHOLD})"
```

Reply:
Good idea